templates/style/base.scss

```diff
@@ -759,3 +759,42 @@ h3 > code,
 h4 > code {
     display: inline-block;
 }
+
+ul.pure-menu-list {
+    li.toggle-source {
+        cursor: pointer;
+        border: 1px solid var(--color-standard);
```

Comment:
`--color-standard` doesn't really fit the theme on ayu/light, they work much better with `--color-border` (dark is better with `--color-standard`, so maybe there needs to be a separate variable for button-borders?).

Reply:
Let's give it a try.